utils.go

```diff
@@ -291,10 +299,21 @@ func toUser(u *pb.User) *User {
 		Nickname: u.Nickname,
 	}
 
-	if u.AvatarLarge != nil && u.AvatarJpg.UrlList != nil {
-		user.ProfilePicture = &ProfilePicture{
-			Urls: u.AvatarJpg.UrlList,
-		}
+	var firstBestImage *pb.Image
+
+	if checkPbImage(u.AvatarLarge) {
+		firstBestImage = u.AvatarLarge
+	} else if checkPbImage(u.AvatarJpg) {
+		firstBestImage = u.AvatarJpg
+	} else if checkPbImage(u.AvatarThumb) {
```

Comment:
Why would thumb be better than avatar medium?

Reply:
If i remember from my tests, medium resolution was 72p and Thumb 90p